lib/prop/index.js

```diff
@@ -1,7 +1,8 @@
-import curry2 from '../_internal/_curry2'
-import view from '../view'
-import lensPath from '../lensPath'
+import _curry2 from '../_internal/_curry2'
 
-export default curry2(function prop(path, obj) {
-  return view(lensPath(path), obj)
+export default _curry2(function prop(path, obj) {
+  if (!Array.isArray(path)) throw new TypeError('Path should be an array')
```

Comment:
Why path should be array? Function should just return object's value at specified key, like
```js
prop('x', { x: 100 }) // 100
```
So, first argument should be `key`, shouldn't it?

This approach is for [path method](http://ramdajs.com/docs/#path)

Reply:
fixed